engine/pkg/meta/model/interfaces.go

```diff
@@ -84,8 +84,9 @@ type ClientConn interface {
 	ClientType() ClientType
 
 	// GetConn gets the underlying connection object
-	// For storeTypeEtcd, it returns *clientv3.Client for the first return param
-	// For storeTypeSQL, it returns *sql.DB
+	// For StoreTypeEtcd, it returns *clientv3.Client for the first return param
+	// For StoreTypeSQL, it returns *sql.DB
+	// For StoreTypeSQLite, it return *gorm.DB
```

Comment:
Why `StoreTypeSQL` and `StoreTypeSQLite` returns different object? Both of them provide SQL interfaces.

Since we have used MySQL for metastore. Can we remove the ETCD implementation of this? Thus we can simplify the code.  e.g. Here we can return a concrete type instead of `interface{}`.

Reply:
1. I will adjust the implementation to reuse the `sql.DB`.
2. Prefer to keep the etcd implementation in case that we want to use `etcd` as business metastore.